app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,38 @@
+class QuestionsController < ApplicationController
+
+  before_filter :load_archive_faq, :except => [:index, :update_positions]
+
+  # GET /archive_faq/:archive_faq_id/questions/manage
+  def manage
+    @questions = @archive_faq.questions.order('position')
+  end
+
+  # fetch archive_faq these questions belong to from db
+  def load_archive_faq
+    @archive_faq = ArchiveFaq.find_by_slug(params[:archive_faq_id])
+    unless @archive_faq.present?
+      flash[:error] = ts("Sorry, we couldn't find the ArchiveFaq you were
```

Comment:
I think as long as we're writing the error message out ourselves, we should use the more natural terminology -- FAQ -- instead of the technical name (ArchiveFAQ).